backend/scripts/sqliteversion.py

```diff
@@ -0,0 +1,22 @@
+import sqlite3
```

Comment:
Nice. Can you rename this file to something like "updatephotographers_sqlite.py"? That way it'll be more clear what it is

Reply:
Done!